nipype/interfaces/utility/base.py

```diff
@@ -128,24 +128,36 @@ class Merge(IOBase):
     def __init__(self, numinputs=0, **inputs):
         super(Merge, self).__init__(**inputs)
         self._numinputs = numinputs
-        add_traits(self.inputs, ['in%d' % (i + 1) for i in range(numinputs)])
+        if numinputs > 0:
+            input_names = ['in%d' % (i + 1) for i in range(numinputs)]
+        elif numinputs == 0:
+            input_names = ['in_lists']
+        else:
```

Comment:
I can do `Merge(1)`. The goal was to leave the interface unmodified for any existing uses, but I guess nobody would actually do `Merge(1)` as it stands.

If we do, I'd suggest making `numinputs` default to 1, so that `Merge()` can be useful.

And I can update the doctests to be demonstrative.